components/infobox/wikis/deadlock/infobox_item_custom.lua

```diff
@@ -0,0 +1,231 @@
+			{name = 'Sprint Speed', parameter = 'sprintspeed'},
+			{name = 'Stamina', parameter = 'Stamina'},
+			{name = 'Slide Distance', parameter = 'slidedistance', funct = '_positivePercentDisplay'},
+			{name = 'Spirit Power', parameter = 'spiritpower'},
+			{name = 'Spirit Resist', parameter = 'spiritresist'},
+			{name = 'Spirit Shield Health', parameter = 'spiritshield'},
+			{name = 'Spirit Lifesteal', parameter = 'spiritlifesteal', funct = '_positivePercentDisplay'},
+			{name = 'Cooldown Reduction', parameter = 'cdreduction', funct = '_positivePercentDisplay'},
+			{name = 'Movement Speed', parameter = 'movespeed'},
+		}
+		widgets = caller:_getAttributeCells(attributeCells)
+		table.insert(widgets, Cell{name = 'Standard Bonus', children = {args.standardbonus}})
+		if not Table.isEmpty(widgets) then
+			table.insert(widgets, 1, Title{children = 'Attributes'})
+		end
+		return widgets
+	elseif id == 'ability' then
+		if String.isEmpty(args.use) and String.isEmpty(args.active) and String.isEmpty(args.passive) then
+			return {}
+		end
+		Array.appendWith(widgets,
+			Cell{name = 'Active', children = {args.active, args.active2, args.active3}},
+			Cell{name = 'Passive', children = {args.passive, args.passive2, args.passive3}}
+		)
+	elseif id == 'availability' then
+		if String.isEmpty(args.category) and String.isEmpty(args.drop) then return {} end
+		return {
+			Title{children = 'Type'},
+			Cell{name = 'Category', children = {caller:_categoryDisplay()}},
+			Cell{name = 'Tier', children = {args.tier}},
+		}
+	elseif id == 'recipe' then
+		if String.isEmpty(args.recipe) then return {} end
+		return {
+			Title{children = 'Components'},
+			Center{children = {args.recipe}}
+		}
+	elseif id == 'maps' then return {}
+	elseif id == 'info' then return {}
+	end
+
+	return widgets
+end
+
+---@param args table
+---@return string[]
+function CustomItem:getWikiCategories(args)
+	if not Namespace.isMain() then return {} end
+
+	local categories = {}
+	if String.isNotEmpty(args.str) or String.isNotEmpty(args.agi) or String.isNotEmpty(args.int) then
+		table.insert(categories, 'Attribute Items')
+	end
+
+	if String.isNotEmpty(args.movespeed) or String.isNotEmpty(args.movespeedmult) then
+		table.insert(categories, 'Movement Speed Items')
+	end
+
+	local possibleCategories = {
+		['Items with Active Abilities'] = 'active',
+		['Items with Passive Abilities'] = 'passive',
+	}
+	for category, requiredArg in pairs(possibleCategories) do
+		if String.isNotEmpty(args[requiredArg]) then
+			table.insert(categories, category)
+		end
+	end
+	return categories
+end
+
+---@param args table
+---@return string?
+function CustomItem.nameDisplay(args)
+	return args.itemname
+end
+
+---@return string
+function CustomItem:_getCostDisplay()
+	local costs = self:getAllArgsForBase(self.args, 'itemcost')
+
+	local innerDiv = CustomItem._costInnerDiv(table.concat(costs, '&nbsp;/&nbsp;'))
```

Comment:
Is there multiple item costs?

Reply:
Technically no, although you could maybe argue the ones for components (E.g if you hadn't bought the component its 9200 souls, with it already bought its 6200)?

Reply:
Will it be used? Otherwise remove